fontbakery-check-ttf.py

```diff
@@ -105,6 +105,33 @@
 NAMEID_LIGHT_BACKGROUND_PALETTE = 23
 NAMEID_DARK_BACKGROUD_PALETTE = 24
 
+NAMEID_STR = {
```

Comment:
But this human-readable text-string is not really different from the constant variable.